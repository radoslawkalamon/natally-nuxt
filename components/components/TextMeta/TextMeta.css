.text-meta {
  color: inherit;
  font-family: 'Tahoma', 'Segoe UI', 'Geneva', 'Verdana', sans-serif;
  font-size: var(--font-size-footnotes);
  line-height: var(--line-height-footnotes);
}
