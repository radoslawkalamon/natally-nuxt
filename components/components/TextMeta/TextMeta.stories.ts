import type { Meta, Story } from '@storybook/vue'
import ComponentsTextMeta from '@/components/components/TextMeta/TextMeta.vue'

const meta: Meta = {
  component: ComponentsTextMeta,
  title: 'Components / Text Meta'
}
export default meta

const Template: Story = (_args, { argTypes }) => ({
  components: {
    ComponentsTextMeta
  },
  props: Object.keys(argTypes),
  template: '<ComponentsTextMeta v-bind="$props" />'
})

export const Default: Story = Template.bind({})
Default.args = {
  text: '28.08.2021'
}
