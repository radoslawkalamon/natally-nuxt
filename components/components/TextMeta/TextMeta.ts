import Vue from 'vue'

export default Vue.extend({
  name: 'ComponentsTextMeta',
  props: {
    text: {
      type: String,
      required: true
    }
  }
})
