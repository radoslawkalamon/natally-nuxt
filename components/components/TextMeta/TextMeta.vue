<template>
  <span
    class="text-meta"
    data-test="components-text-meta"
    v-text="text"
  />
</template>

<script src="./TextMeta.ts" lang="ts"></script>
<style src="./TextMeta.css" scoped></style>
