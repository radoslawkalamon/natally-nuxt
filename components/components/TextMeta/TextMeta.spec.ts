import TextMeta from '@/components/components/TextMeta/TextMeta.vue'
import { shallPassUnitSanityTest } from '@/devtools/jest.common.spec'

describe('Wrappers / Text Meta', () => {
  describe('Unit', () => {
    shallPassUnitSanityTest({
      component: TextMeta,
      options: {
        propsData: {
          text: 'Default Prop Text'
        }
      }
    })
  })
})
