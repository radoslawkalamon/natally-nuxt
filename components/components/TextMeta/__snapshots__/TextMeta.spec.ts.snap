// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrappers / Text Meta Unit shall pass unit sanity test 1`] = `"<p data-test=\\"components-text-meta\\" class=\\"text-meta\\">Default Prop Text</p>"`;
